wolfcrypt/src/random.c

```diff
@@ -2639,8 +2639,17 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         {
             /* may still have /dev/random */
             os->fd = open("/dev/random", O_RDONLY);
-            if (os->fd == -1)
+            if (os->fd == -1) {
```

Comment:
With DOLPHIN_EMULATOR will os->fd always be -1 here?  If so, it may be better to put the block below in to a new wc_GenerateSeed() implementation specific to DOLPHIN_EMULATOR.

Reply:
You are correct. I rewrote it as a separate implementation of `wc_GenerateSeed`.

Reply:
Thanks!